Woof/Woof/View/ReusableViews/CopyToClipboardLabel.swift

```diff
@@ -0,0 +1,41 @@
+import SwiftUI
+
+/// A label for user interface items, consisting of an icon with two docs and with title `Copy`.
+struct CopyToClipboardLabel: View {
+    // MARK: - Internal interface
+
+    var body: some View {
+        Label {
+            Text(labelText)
+        } icon: {
+            Image(systemName: String.IconName.doc)
+        }
+    }
+
+    // MARK: - Private interface
+
+    private let labelText = "Copy"
+}
+
+struct CopyToClipboardLabel_Previews: PreviewProvider {
+    static var previews: some View {
+        VStack {
+            CopyToClipboardLabel()
+
+            Divider()
+
+            CopyToClipboardLabel()
+                .foregroundColor(.blue)
+                .font(.system(.subheadline))
+
+            Divider()
+
+            CopyToClipboardLabel()
+                .foregroundColor(.white)
+                .font(.system(.title))
+                .padding()
+                .background(Color.black)
+                .clipShape(RoundedRectangle(cornerRadius: 10))
```

Comment:
I don't get the idea why we need all these variants in the preview, could you please explain?

Reply:
To provide quick and visual representation of the label and customise capability.

Reply:
It looks like you violated the YAGNI principle.

Reply:
agree with @iBYarchitect , it's unnecessary code for me, your view `CopyToClipboardLabel` conform to the `View` protocol, so it is expected behavior.

Reply:
Fixed